apps/meteor/ee/server/apps/communication/endpoints/appsCountHandler.ts

```diff
@@ -0,0 +1,27 @@
+import type { AppManager } from '@rocket.chat/apps-engine/server/AppManager';
+
+import { API } from '../../../../../app/api/server';
+import type { AppsRestApi } from '../rest';
+import { getAppsConfig } from '../../../../app/license/server/license';
+
+export const appsCountHandler = (apiManager: AppsRestApi) =>
```

Comment:
is this a new way to declare endpoints?

Reply:
@d-gubert

Reply:
Endpoints for apps have been defined inside the `[...]/communication/rest.js` file for a long time, which caused the file to become huge. I've simply extracted the handler definition to another file. The path definition for the endpoint is still in the `rest.js` file, though